packages/fxa-settings/src/components/ChooseNewsletters/index.test.tsx

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
+import { FluentBundle } from '@fluent/bundle';
+import { Subject } from './mocks';
+
+describe('ChooseNewsletters component', () => {
+  let bundle: FluentBundle;
+  beforeAll(async () => {
+    bundle = await getFtlBundle('settings');
+  });
+  it('renders newsletter options as expected', async () => {
+    render(<Subject />);
+    testAllL10n(screen, bundle);
+
+    screen.getByText(
+      'Practical knowledge is coming to your inbox. Sign up for more:'
+    );
+
+    const checkboxes = await screen.findAllByRole('checkbox');
+    expect(checkboxes).toHaveLength(3);
```

Comment:
Instead of hard coding a magic `3` do we want to assert the against the number of newsletters exported from `packages/fxa-settings/src/components/ChooseNewsletters/newsletters.ts`? (or wherever that 3 is comin' from)